### PR TITLE
fix(gatsby): update terser-webpack-plugin to avoid webpack error

### PR DESCRIPTION
**Pull request by sidharthachatterjee**
`terser-webpack-plugin` broke two days ago because a minor version bump to `terser` had a breaking change

This was fixed in https://github.com/terser-js/terser/commit/03b08760fcac9ee3f633d7c8e82af417c47db4e7 and released in 

This PR updates dependencies and lock files for starters so that no one accidentally installs the breaking version again

Related to https://github.com/gatsbyjs/gatsby/issues/11508

### CODE REVIEW

File: starters/blog/package-lock.json [hunk 1]
```diff
@@ -891,7 +891,7 @@
     },
     "@types/debug": {
       "version": "0.0.29",
-      "resolved": "https://registry.npmjs.org/@types/debug/-/debug-0.0.29.tgz",
+      "resolved": "http://registry.npmjs.org/@types/debug/-/debug-0.0.29.tgz",
```
Comment (DSchau) [inline starters/blog/package-lock.json]: Hmm.. wonder why this change?

**Comment by sidharthachatterjee** [comment]
diff for starters is broken because of an npm issue 

https://npm.community/t/npm-install-downgrading-resolved-packages-from-https-to-http-registry-in-package-lock-json/1818/14

Recommendation is to manually update http back to https, let me do that

**Comment by DSchau** [comment]
@sidharthachatterjee if you want the fix instead -> https://npm.community/t/some-packages-have-dist-tarball-as-http-and-not-https/285/56

**Comment by pieh** [comment]
Should we just bump `terser-webpack-plugin` here -> publish and let renovate handle bumps in starters?

**Comment by sidharthachatterjee** [comment]
@pieh That seems like a better idea, reverting changes to starter locks

File: starters/blog/package-lock.json (hunk 1 above)
Comment (sidharthachatterjee) [reply starters/blog/package-lock.json]: Only updating gatsby dependency now and letting Renovate handle updates to starters

**Comment by sidharthachatterjee** [comment]
We're good now, @pieh

**Comment by DSchau** [comment]
Published as [`gatsby@2.0.112`](https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby/CHANGELOG.md#20112-2019-02-04)
